staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -294,20 +299,28 @@ func (o *DeleteOptions) RunDelete(f cmdutil.Factory) error {
 }
 
 func (o *DeleteOptions) DeleteResult(r *resource.Result) error {
-	found := 0
 	if o.IgnoreNotFound {
 		r = r.IgnoreErrors(errors.IsNotFound)
 	}
+
+	deletedInfos, err := r.Infos()
+	if len(deletedInfos) == 0 {
+		fmt.Fprintf(o.Out, "No resources found\n")
+		return err
+	}
+	if o.Interactive && !o.confirmation(deletedInfos) {
+		fmt.Fprintf(o.Out, "deletion is cancelled\n")
+		return err
+	}
+	var errs []error
+	if err != nil {
+		errs = append(errs, err)
```

Comment:
Same question about what this error might be, but in this case I'm wondering why we should proceed with the deletion despite whatever this error was